### PR TITLE
Docs: modified last gpexpand step and added steps for pxf

### PR DESCRIPTION
**Pull request by mperezfuster**
I have renamed the last step "Removing the Expansion Schema" to "Post Expansion Tasks" and added steps for initializing the new PXF hosts.
This should work as a placeholder for when we document post expansion steps for other components (GPSS, GPCC).
I am assuming that the PXF hosts can be initialized after the gpexpand process is fully completed but please correct me if this is not right.

Preview:
https://mireia-gpexpand-pxf.sc2-04-pcf1-apps.oc.vmware.com/7-0/admin_guide/expand/expand-post.html

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/expand/expand-post.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        <li id="no163298">Log in on the master host as the user who will be running your Greenplum
+          Database system (for example, <codeph>gpadmin</codeph>).</li>
+        <li id="no163294">Run the <codeph>gpexpand</codeph> utility with the <codeph>-c</codeph>
+          option. For example:<codeblock>$ gpexpand -c</codeblock><note type="note">Some systems
+            require you to press Enter twice.</note></li>
+      </ol>
+    </body>
+  </topic>
+  <topic id="topic_pxl_1q2_hpb">
+    <title>Initializing new PXF segments</title>
```
Comment (lisakowen) [inline gpdb-doc/dita/admin_guide/expand/expand-post.xml:31]: initialize is a general term, but it does have specific connotations for pxf 5 that are not required in pxf 6.  wondering if  something a little more generic likd "Setting Up PXF on the New Host" might work?

and i think both titles should be using caps, i.e. "Removing the Expansion Schema", whatever this one is.

File: gpdb-doc/dita/admin_guide/expand/expand-post.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    <body>
+      <p>If you are using PXF in your Grenplum Database cluster, you must perform some configuration
+        steps on the new hosts.</p>
+      <p>There are different steps to follow depending on your PXF version and type of
```
Comment (lisakowen) [inline gpdb-doc/dita/admin_guide/expand/expand-post.xml:35]: ... version and *the* type of ...   or  and the source of

File: gpdb-doc/dita/admin_guide/expand/expand-post.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        installation.</p>
+      <section>
+        <title>PXF 5</title>
+        <ul id="ul_u44_r22_hpb">
+          <li>If you installed the PXF 5.x <codeph>rpm</codeph> on your Greenplum cluster, you must
+                <codeph><xref href="../../../pxf/5-16/installing_pxf.html">install</xref></codeph>
+            the same version on the new hosts.</li>
+          <li>Once PXF is installed on the new segment hosts, log into the Greenplum Master and run
+            the following commands:<codeblock>pxf cluster reset
+plx cluster init</codeblock></li>
```
Comment (lisakowen) [inline gpdb-doc/dita/admin_guide/expand/expand-post.xml:45]: plx -> pxf (all occurrences)

File: gpdb-doc/dita/admin_guide/expand/expand-post.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    <title>Initializing new PXF segments</title>
+    <body>
+      <p>If you are using PXF in your Grenplum Database cluster, you must perform some configuration
+        steps on the new hosts.</p>
+      <p>There are different steps to follow depending on your PXF version and type of
+        installation.</p>
+      <section>
+        <title>PXF 5</title>
+        <ul id="ul_u44_r22_hpb">
+          <li>If you installed the PXF 5.x <codeph>rpm</codeph> on your Greenplum cluster, you must
```
Comment (lisakowen) [inline gpdb-doc/dita/admin_guide/expand/expand-post.xml:40]: i wonder if we should also explicitly state something to address those customers using the PXF distributed with greenplum ...  "if you are using the PXF installed/included with your greenplum distribution, no install action is required on your part."

File: gpdb-doc/dita/admin_guide/expand/expand-post.xml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+                <codeph><xref href="../../../pxf/5-16/installing_pxf.html">install</xref></codeph>
+            the same version on the new hosts.</li>
+          <li>Once PXF is installed on the new segment hosts, log into the Greenplum Master and run
+            the following commands:<codeblock>pxf cluster reset
```
Comment (lisakowen) [inline gpdb-doc/dita/admin_guide/expand/expand-post.xml:44]: in the pxf docs, we usually preface commands with something like the following to reinforce the user and host that the command should be run on:
```
gpadmin@gpmaster$
```
this isn't a convention in the greenplum docs, and commands in the section above aren't doing it, so maybe just added a $ in front of the command should suffice.

File: gpdb-doc/dita/admin_guide/expand/expand-post.xml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+plx cluster init</codeblock></li>
+        </ul>
+      </section>
+      <section>
+        <title>PXF 6</title>
+        <ul id="ul_xwz_wc2_hpb">
+          <li>If you installed the PXF 6.x <codeph>rpm</codeph> on your Greenplum cluster, you must
```
Comment (lisakowen) [inline gpdb-doc/dita/admin_guide/expand/expand-post.xml:51]: rpm -> rpm or deb

**Review by lisakowen — CHANGES_REQUESTED** [review]
thanks, i just had a few minor comments.

(this is a great reorganization, it will be easy to add expansion-related sections for other components.)

File: gpdb-doc/dita/admin_guide/expand/expand-post.xml [hunk 7]
```diff
@@ -28,33 +28,36 @@
...
-                <codeph><xref href="../../../pxf/6-0/installing_pxf.html">install</xref></codeph>
-            the same version on the new hosts.</li>
-          <li>Once PXF is installed on the new segment hosts, log into the Greenplum Master and run
-            the following commands:
-            <codeblock>pxf cluster register
-plx cluster sync</codeblock></li>
+          <li>If you installed the PXF 6.x <codeph>rpm</codeph> or <codeph>deb</codeph> on your
+            Greenplum cluster, you must <codeph><xref href="../../../pxf/6-0/installing_pxf.html"
+                >install</xref></codeph> the same version on the new hosts.</li>
+          <li>If you are using the PXF version included with your Greenplum Database distribution,
```
Comment (lisakowen) [inline gpdb-doc/dita/admin_guide/expand/expand-post.xml:56]: pxf 6 is not embedded in greenplum, so i think we can remove this bullet.

**Review by lisakowen — APPROVED** [review]
looks good, please remove the noted statement in the pxf  6 section.

**Review by bradfordb-vmware — APPROVED** [review]

**Review by ashuka24 — APPROVED** [review]

File: gpdb-doc/dita/admin_guide/expand/expand-post.xml [hunk 8]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic
+  PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
+<topic id="expand-rm-schema" xml:lang="en">
+  <title id="no163283">Post Expansion Tasks</title>
+  <shortdesc>Once the expansion is completed, you must perform different tasks depending on your
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/expand/expand-post.xml:6]: Once -> After

File: gpdb-doc/dita/admin_guide/expand/expand-post.xml [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+    environment.</shortdesc>
+  <body>
+    <ul id="ul_fhs_yb2_hpb">
+      <li><xref href="#topic_xvp_5p2_hpb" format="dita"/></li>
+      <li><xref href="#topic_pxl_1q2_hpb" format="dita"/></li>
+    </ul>
+  </body>
+  <topic id="topic_xvp_5p2_hpb">
+    <title>Removing the expansion schema</title>
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/expand/expand-post.xml:15]: Init caps:  Expansion Schema

File: gpdb-doc/dita/admin_guide/expand/expand-post.xml [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+    <body>
+      <p>To clean up after expanding the Greenplum cluster, remove the expansion schema.</p>
+      <p>You can safely remove the expansion schema after the expansion operation is complete and
+        verified. To run another expansion operation on a Greenplum system, first remove the
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/expand/expand-post.xml:19]: Maybe reword to:  You _must_ remove the existing expansion schema before you can perform another expansion operation on the Greenplum system.

File: gpdb-doc/dita/admin_guide/expand/expand-post.xml [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+        <li id="no163294">Run the <codeph>gpexpand</codeph> utility with the <codeph>-c</codeph>
+          option. For example:<codeblock>$ gpexpand -c</codeblock><note type="note">Some systems
+            require you to press Enter twice.</note></li>
+      </ol>
+    </body>
+  </topic>
+  <topic id="topic_pxl_1q2_hpb">
+    <title>Setting Up PXF on the New Host</title>
+    <body>
+      <p>If you are using PXF in your Grenplum Database cluster, you must perform some configuration
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/expand/expand-post.xml:33]: typo: Grenplum

**Review by dyozie — CHANGES_REQUESTED** [review]

**Review by dyozie — APPROVED** [review]
